src/pages/project/CreateProjectPage.tsx

```diff
@@ -1,47 +1,109 @@
+
+    const call = async () => {
+      const createProjectResult = await createProject();
+
+      if (createProjectResult.data?.createProject?.result?.__typename === 'Project') {
+        const projectId = createProjectResult.data.createProject?.result?.vid || undefined;
+
+        localStorage.setItem(BLANK_PROJECT_ID, String(projectId));
+
+        setBlankProjectId(projectId);
+      }
+
+      if (createProjectResult.data?.createProject?.result?.__typename === 'Error') {
+        const inlineCreateProjectError = createProjectResult.data?.createProject?.result;
+
+        // TODO: Обработать внутреннюю ошибку при создании проекта
+
+        // eslint-disable-next-line no-console
+        console.log('inlineCreateProjectError', inlineCreateProjectError);
+      }
+    };
+
+    if (data) {
+      setBlankProjectId(data);
+    } else {
+      call();
+    }
+  }, [createProject]);
+
+  const {
+    data: queryRegionListData,
+    loading: queryRegionListLoading,
+    error: queryRegionListError,
+  } = useQueryRegionList();
 
   const referenceData: ReferenceDataType = { regionList: queryRegionListData?.regionList };
 
-  const [
-    createProject,
-    { data: createProjectData, called: isCreateProjectCalled },
-  ] = useCreateProject({
-    onCompleted() {
-      history.push('/projects');
-    },
-  });
-
-  const handleFormSubmit = (values: CreateProjectVariables) => {
-    createProject({ variables: values });
+  const handleFormSubmit = async (values: FormValues) => {
+    const updateProjectResult = await updateProject({
+      variables: {
+        vid: blankProjectId,
+        name: values.description.name,
+        type: values.description.type,
+        region: values.description.region,
+        coordinates: values.description.coordinates,
+        description: values.description.description,
+        yearStart: values.description.yearStart,
+        status: ProjectStatusEnum.Unpublished,
+        version: 1,
+      },
+    });
+
+    if (updateProjectResult.data?.updateProject?.result?.__typename === 'Project') {
+      const projectId = updateProjectResult.data.updateProject?.result?.vid || undefined;
+
+      localStorage.removeItem(BLANK_PROJECT_ID);
+
+      history.push(`/projects/show/${projectId}`);
+    }
+
+    if (updateProjectResult.data?.updateProject?.result?.__typename === 'Error') {
+      const inlineCreateProjectError = updateProjectResult.data?.updateProject?.result;
+
+      // TODO: Обработать внутреннюю ошибку при создании проекта
+
+      // eslint-disable-next-line no-console
+      console.log('inlineCreateProjectError', inlineCreateProjectError);
+    }
   };
 
-  if (isCreateProjectCalled && createProjectData) {
+  if (createProjectError || updateProjectError || queryRegionListError) {
     // eslint-disable-next-line no-console
-    console.log({ createProjectData });
+    console.log(createProjectError, updateProjectError, queryRegionListError);
```

Comment:
```suggestion
    console.log({ createProjectError, updateProjectError, queryRegionListError });
```
можно так сделать, тогда в консоли будет проще понять какая именно ошибка произошла, а какие нет

Reply:
Исправил